packages/fal/src/fal-image-model.ts

```diff
@@ -92,6 +109,34 @@ export class FalImageModel implements ImageModelV2 {
         timestamp: currentDate,
         headers: responseHeaders,
       },
+      providerMetadata: {
+        fal: {
+          images: targetImages.map((image, index) => {
+            const {
+              url,
+              content_type: contentType,
+              file_name: fileName,
+              file_data: fileData,
+              file_size: fileSize,
+              ...imageMetaData
+            } = image;
+
+            const nsfw =
+              value.has_nsfw_concepts?.[index] ??
+              value.nsfw_content_detected?.[index];
+
+            return {
+              ...imageMetaData,
+              ...(contentType !== undefined ? { contentType } : undefined),
+              ...(fileName !== undefined ? { fileName } : undefined),
+              ...(fileData !== undefined ? { fileData } : undefined),
+              ...(fileSize !== undefined ? { fileSize } : undefined),
```

Comment:
I normalized some of the `providerMetaData` properties that I think the way we normalized `revised_prompt` to `revisedPrompt` in the Open AI Image model. 

Shall I also camelize `debug_latents`, `debug_per_pass_latents`, and their respective properties?

Reply:
wonder how far we should to here. usually we do this but it is also overhead

Reply:
> wonder how far we should to here. usually we do this but it is also overhead 

I agree. I think we should do it for properties that we see repeatedly across providers and models. But e.g. not go overboard with things like `debug_latents` which is specific to a single model from `fal`

I also wonder if we should pass through provider options that we don't handle ourselves. Otherwise we will always play catch up whenever some model from some provider exposes a new information. We could do it in a way that clearly communicates that we don't take responsibility, like `.providerMetaData.fal.unsafe_otherData`  or similar.